build.gradle

```diff
@@ -46,3 +47,15 @@ publishPlugin {
     token = System.getenv("ORG_GRADLE_PROJECT_intellijPublishToken")
     channels 'beta'
 }
+
+checkstyle {
+    configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
+    toolVersion = 8.35
+}
+
+tasks.withType(Checkstyle) {
+    reports {
+        xml.enabled false
+        html.enabled true
```

Comment:
How can we access this report?

Reply:
When `./gradlew build` is running, The checkstyle is checking and showing as below. There is no need to access the report actually.

![image](https://user-images.githubusercontent.com/16450962/91408626-1b9d3680-e87f-11ea-8437-4ed173ab5866.png)

Reply:
And you can show the cause of failure in github action
 - https://github.com/naver/openwhisk-intellij-plugin/runs/1019621549?check_suite_focus=true
![image](https://user-images.githubusercontent.com/16450962/91408780-6454ef80-e87f-11ea-928f-b68593615077.png)

Reply:
ok I just asked because it is enabled.